---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

{% if author.googlescholar %}
  You can also find my articles on  my <u><a href="{{author.googlescholar}}">Google Scholar</a></u> profile.
{% endif %}

You can also find my articles on my <u><a href="https://scholar.google.com/citations?user=t1a1AwUAAAAJ&hl=en">Google Scholar</a></u> profile.

{% include base_path %}

<!---
{% for post in site.publications reversed %}
  {% include archive-single.html %}
{% endfor %}
--->
<ol>

<li> <b>Li, G.*</b>, Luan, C.*, Zentz, S., Zelt, R., Roach, J., Liu, J., Qian, L., Li, Y., Yang, Y. (2021). <a href="https://www.mdpi.com/1422-0067/22/16/8943"> ExpressHeart: Web Portal to Visualize Transcriptome Profiles of Non-cardiomyocyte Cells.</a>  International Journal of Molecular Sciences, 2021; 22(16):8943.  [<a href="http://shiny.bios.unc.edu/expressheart/"> Web </a>] </li>

<li> Yang,  Y., <b>Li,  G.</b>,  Xie,  Y.,  Wang,  L.,  Liu,  J.,  Qian,  L.,  Li,  Y.  (2021). <a href="https://doi.org/10.1093/bib/bbab122"> iSMNN: Batch   Effect   Correction   for   Single-cell   RNA-seq   data   via   Iterative   Supervised   Mu-tual   Nearest   Neighbor   Refinement.</a>
 Briefings   in   Bioinformatics,   2021;,   bbab122  [<a href="http://shiny.bios.unc.edu/expressheart/"> Code </a>] </li>


<li> <b>Li,  G.</b>,  Raffield,  L.,  Logue,  M.,  Miller,  M.W.,  Santos,  H.P.,  Oâ€™Shea,  M.,  Fry,  R. and Li,Y. (2020). <a href="https://doi.org/10.1080/15592294.2020.1827716"> CUE: CpG imPutation Ensemble for DNA Methylation Levels across Platforms.</a> Epigenetics, 1-11. [<a href="https://github.com/GangLiTarheel/CUE"> Code </a>]  </li>


<li> <b>Li,   G.</b>,   Hannig,   J.   (2020). <a href="https://doi.org/10.1002/sta4.308"> Deep   Fiducial   Inference.</a> Stat. 2020; e308. [<a href="https://github.com/GangLiTarheel/FAE"> Code </a> ] </li>


<li> Yang, Y.*, <b>Li, G.*</b>, Qian, H., Wilhelmsen, K., Shen, Y., Li., Y. (2020). [<a href="https://doi.org/10.1093/bib/bbaa097">SMNN: Batch Effect Correction for Single-cell RNA-seq data via Supervised Mutual Nearest Neighbor Detection.</a> Briefings in Bioinformatics, Volume 22, Issue 3, May 2021, bbaa097, <a href="https://github.com/yycunc/iSMNN"> Code </a>]
</li>


<li> <b>Li,  G.</b>,  Yang,  Y.,  Van  Buren,  E.,  Li,  Y.  (2019).  <a href="https://journals.lww.com/jbioxresearch/Fulltext/2019/12000/Dropout_imputation_and_batch_effect_correction_for.4.aspx">Dropout  imputation  and  batch  effectcorrection for single-cell RNA sequencing data.</a> Journal of Bio-X Research, 2(4), 169-177. </li>

</ol>

Note:*means co-first author.  

<h1>Dissertation</h1>
<ul>

<li> <a href="https://www.proquest.com/docview/2546056928/fulltextPDF/7C7D9A9F06404D64PQ">Coupling Machine Learning with Fiducial Inference, Genetics and Epigenetics</a>. Ann Arbor: The University of North Carolina at Chapel Hill; 2021. </li>
</ul>

<h1>Book Chapter</h1>
<ol>

<li> Wu, D., Karhade, D. S., Pillai, M., Jiang, M. Z., Huang, L., <b>Li, G.</b>, ... & Divaris, K. (2021). <a href="https://link.springer.com/chapter/10.1007/978-3-030-71881-7_13">Machine Learning and Deep Learning in Genetics and Genomics.</a>  In Machine Learning in Dentistry (pp. 163-181). Springer, Cham. </li>

</ol>
