---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

{% if author.googlescholar %}
  You can also find my articles on  my <u><a href="{{author.googlescholar}}">Google Scholar</a></u> profile.
{% endif %}

You can also find my articles on my <u><a href="https://scholar.google.com/citations?user=t1a1AwUAAAAJ&hl=en">Google Scholar</a></u> profile.

{% include base_path %}

<!---
{% for post in site.publications reversed %}
  {% include archive-single.html %}
{% endfor %}
--->


<h1>Work in progress</h1>
<ol>

<li> <b>Li, G.*</b>, Eva K. Nichols*, Valentino E. Browning, Nicolas Longhi,  Conor K. Camplisson, Beliveau, B.,  Noble, W. (2024+). 
	<a href="https://www.biorxiv.org/content/10.1101/2024.08.30.610553v2">Predicting cell cycle stage from 3D single-cell nuclear-stained images.  </a> 
  <a href="https://github.com/Noble-Lab/cellcyclenet"> [Code] </a> 
  <a href="https://www.youtube.com/watch?v=GflTIcI5inA"> [Presentation] </a>
  </li>

<li> <b>Li, G.</b>, Kim, H., Pendyala, S., Zhang, R., Disteche, C., Vert, J., Deng, X., Fowler, D., Noble, W.(2024+, submitted). 
  <a href="https://www.biorxiv.org/content/10.1101/2023.11.03.565575v1.article-info">Pseudotime analysis and integration for time-series single-cell sequencing and imaging data. </a> 
  <a href="https://github.com/Noble-Lab/Sceptic"> [Code] </a> 
  <a href="https://www.youtube.com/watch?v=Ggv5k4Fubbs"> [Presentation] </a>
  </li>



<li> <b>Li, G.</b>, Chengxiang Qiu, Xumeng Zhang, Kris G. Alavattam, Beth Martin, Josephine Lin, Riza M. Daza, Galina N. Filippova, Ran Zhang, Jay Shendure, Xinxian Deng, Disteche M. Christine, William S. Noble. (2024+).
    Allelic change surveyed by high throughput single-cell omics during mouse embryonic development. </li>


<li> <b>Li, G.</b>, Kim, H., Zhang, R., Pendyala, S., Papaxanthos, L., Deng, X., Disteche, C., Fowler, D., Vert, J., Noble, W.(2024+). 
	Integrating imaging and sequencing data from visual cell sorting. </li>

<li> <b>Li, G.*</b>, Jia, W.*, Chen, J., Sun, Q., Liu, W., Guan, W., Lai, B., He, X., Szatkiewicz, J., Sullivan, P. , Li, Y.(2024+, submitted). 
	<a href="https://www.biorxiv.org/content/10.1101/2022.12.20.521277v1">DeepGWAS to Enhance GWAS Signals for Neuropsychiatric Disorders via Deep Neural Network.</a>  <a href="https://github.com/GangLiTarheel/DeepGWAS"> [Code] </a>
  </li>


<li>Ran Zhang, Chengxiang Qiu, Gala Filippova, <b>Gang Li</b>, Jay Shendure, Jean-Philippe Vert, Xinxian Deng, Christine M. Disteche, and William Stafford Noble. (2024+) 
<a href="https://pubmed.ncbi.nlm.nih.gov/38496477/">Multi-condition and multi-modal temporal profile inference during mouse embryonic development.</a>  
 </li>

<li> Chenwei Tang, Quan Sun, Xinyue Zeng, Xiaoyu Yang, Fei Liu,  Jinying Zhao, <b>Gang Li</b>, Yin Shen, Bixiang Liu, Jia Wen, Yun Li  (2024+).
  <a href="https://www.biorxiv.org/content/10.1101/2024.05.23.595514v1">Cell-type specific inference from bulk RNA-sequencing data by integrating single cell reference profiles via EPIC-unmix.</a>
   </li>


</ol>
Note: * means co-first author.  

<h1>Publications as leading author</h1>
<ol>

<li> <b>Li, G.*</b>, Luan, C.*, Zentz, S., Zelt, R., Roach, J., Liu, J., Qian, L., Li, Y., Yang, Y. (2021). <a href="https://www.mdpi.com/1422-0067/22/16/8943"> ExpressHeart: Web Portal to Visualize Transcriptome Profiles of Non-cardiomyocyte Cells.</a>  International Journal of Molecular Sciences, 2021; 22(16):8943.  <a href="http://shiny.bios.unc.edu/expressheart/"> [Web] </a> </li>

<li> <b>Li,  G.</b>,  Raffield,  L.,  Logue,  M.,  Miller,  M.W.,  Santos,  H.P.,  O’Shea,  M.,  Fry,  R. and Li,Y. (2020). <a href="https://doi.org/10.1080/15592294.2020.1827716"> CUE: CpG imPutation Ensemble for DNA Methylation Levels across Platforms.</a> Epigenetics, 1-11. <a href="https://github.com/GangLiTarheel/CUE"> [Code] </a>  </li>


<li> <b>Li,   G.</b>,   Hannig,   J.   (2020). <a href="https://doi.org/10.1002/sta4.308"> Deep   Fiducial   Inference.</a> Stat. 2020; e308. <a href="https://github.com/GangLiTarheel/FAE"> [Code] </a>  </li>


<li> Yang, Y.*, <b>Li, G.*</b>, Qian, H., Wilhelmsen, K., Shen, Y., Li., Y. (2020). <a href="https://doi.org/10.1093/bib/bbaa097">SMNN: Batch Effect Correction for Single-cell RNA-seq data via Supervised Mutual Nearest Neighbor Detection.</a> Briefings in Bioinformatics, Volume 22, Issue 3, May 2021, bbaa097, <a href="https://github.com/yycunc/SMNN"> [Code] </a>
</li>


<li> <b>Li,  G.</b>,  Yang,  Y.,  Van  Buren,  E.,  Li,  Y.  (2019).  <a href="https://journals.lww.com/jbioxresearch/Fulltext/2019/12000/Dropout_imputation_and_batch_effect_correction_for.4.aspx">Dropout  imputation  and  batch  effectcorrection for single-cell RNA sequencing data.</a> Journal of Bio-X Research, 2(4), 169-177. </li>

</ol>



<h1>Collaborative paper</h1>
<ol>

<li>  Tangqi Fang, Yifeng Liu, Addie Woicik, Minsi Lu, Anupama Jha, Xiao Wang, <b>Gang Li</b>, Borislav Hristov, Zixuan Liu, Hanwen Xu, William Noble, Sheng Wang. (2024 ISMB accepted). <a href="https://www.biorxiv.org/content/10.1101/2023.10.25.564065v2">Enhancing Hi-C contact matrices for loop detection with Capricorn, a multi-view diffusion model.</a>  <a href="https://github.com/CHNFTQ/Capricorn"> [Code]  </a>
 </li>

<li>  Gao W, Gu K, Ma L, Yang F, Deng L, Zhang Y, Miao MZ, Li W, <b>Li G</b>, Qian H, Zhang Z, Wang G, Yu H, Liu X. (2024).
    <a href="https://www.ijbs.com/v20p2727.htm">Interstitial Fluid Shear Stress Induces the Synthetic Phenotype Switching of VSMCs to Release Pro-calcified Extracellular Vesicles via EGFR-MAPK-KLF5 Pathway.</a> Int J Biol Sci 2024; 20(7):2727-2747. doi:10.7150/ijbs.90725. https://www.ijbs.com/v20p2727.htm
    </li> 

<li>  Y. Yang, T. Alves, M. Z. Miao, Y.C. Wu, <b>G. Li</b>, J. Lou, H. Hasturk, T.E. Van Dyke, A. Kantarci, D. Wu (2023).
    <a href="https://journals.sagepub.com/doi/10.1177/00220345231205283">Single-Cell Transcriptomic Analysis of Dental Pulp and Periodontal Ligament Stem Cells.</a> Journal of Dental Research. 2023;0(0). doi:10.1177/00220345231205283
    </li> 

<li>  Michael Z. Miao, Qian Peter Su, Yang Cui, Edward M. Bahnson, <b>Gang Li</b>, Menglin Wang, Yuchen Yang, John A. Collins1, Di Wu, Susan Chubinskaya, Brian O. Diekman, Richard F. Loeser (2023).
    <a href="https://www.science.org/doi/10.1126/scisignal.adf8299">Redox-active endosomes mediate α5β1 integrin signaling and promote chondrocyte matrix metalloproteinase production in osteoarthritis.</a> Sci. Signal.16,eadf8299(2023).DOI:10.1126/scisignal.adf8299
    </li> 

<li>  Laetitia Meng-Papaxanthos, Ran Zhang, <b>Li, G.</b>, Marco Cuturi, William Stafford Noble, Jean-Philippe Vert (2023).
    <a href="https://doi.org/10.1093/bioinformatics/btad420">LSMMD-MA: scaling multimodal data integration for single-cell genomics data analysis.</a> , Bioinformatics, Volume 39, Issue 7, July 2023, btad420, https://doi.org/10.1093/bioinformatics/btad420 <a href="https://pypi.org/project/lsmmdma/"> [Code] </a>
    </li> 

<li>   Le Huang, Yuchen Yang, <b>Gang Li</b>, Minzhi Jiang, Jia Wen, Armen Abnousi, Jonathan D Rosen, Ming Hu, Yun Li. (2022).
    <a href="https://doi.org/10.1093/bib/bbac145"> A systematic evaluation of Hi-C data enhancement methods for enhancing PLAC-seq and HiChIP data.</a>  Briefings in Bioinformatics, 2022;, bbac145, https://doi.org/10.1093/bib/bbac145 </li> 

<li> Yu, W., Gu, Q., Wu, D., Zhang, W., <b>Li, G.</b>, ..., Lu, E. (2022). 
	<a href="https://onlinelibrary.wiley.com/doi/10.1111/jre.12989">Identification of potentially functional circRNAs and prediction of circRNA-miRNA-mRNA regulatory network in periodontitis: bridging the gap between bioinformatics and clinical needs.</a>
  J Periodontal Res. 2022 Apr 6. doi: 10.1111/jre.12989. Epub ahead of print. PMID: 35388494. </li>

<li> Ren, X., Wang, M., ..., <b>Li, G.</b>, ..., Shen, Y. (2021). <a href="https://www.science.org/doi/10.1126/sciadv.abi4360"> Parallel characterization of cis-regulatory elements for multiple genes using CRISPRpath.  </a> Science Advances. 2021 Sep;7(38):eabi4360. DOI: 10.1126/sciadv.abi4360. PMID: 34524848; PMCID: PMC8443183. </li>

<li> Yang,  Y., <b>Li,  G.</b>,  Xie,  Y.,  Wang,  L.,  Liu,  J.,  Qian,  L.,  Li,  Y.  (2021). <a href="https://doi.org/10.1093/bib/bbab122"> iSMNN: Batch   Effect   Correction   for   Single-cell   RNA-seq   data   via   Iterative   Supervised   Mu-tual   Nearest   Neighbor   Refinement.</a>
 Briefings   in   Bioinformatics,   2021;,   bbab122  <a href="https://github.com/yycunc/iSMNN"> [Code] </a> </li>

</ol>

<h1>Book Chapter</h1>
<ol>

<li> <b>Li, G.</b>, Zhang, G., Li, Y. (2022).	
		DNA Methylation Imputation across Platforms. <a href="https://www.barnesandnoble.com/w/epigenome-wide-association-studies-weihua-guan/1139992489">Epigenome-Wide Association Studies: Methods and Protocols.</a> Springer, US, 2022. Methods in Molecular Biology \#2432.
    </li>

<li> Wu, D., Karhade, D. S., Pillai, M., Jiang, M. Z., Huang, L., <b>Li, G.</b>, ... & Divaris, K. (2021). <a href="https://link.springer.com/chapter/10.1007/978-3-030-71881-7_13">Machine Learning and Deep Learning in Genetics and Genomics.</a>  In Machine Learning in Dentistry (pp. 163-181). Springer, Cham. </li>

</ol>

<h1>Dissertation</h1>
<ul>

<li> <a href="https://www.proquest.com/docview/2546056928/fulltextPDF/7C7D9A9F06404D64PQ">Coupling Machine Learning with Fiducial Inference, Genetics and Epigenetics</a>. Ann Arbor: The University of North Carolina at Chapel Hill; 2021. </li>
</ul>